src/vmm/src/devices/virtio/mmio.rs

```diff
@@ -239,7 +245,13 @@ impl MmioTransport {
                     }
                     0x34 => self.with_queue(0, |q| u32::from(q.get_max_size())),
                     0x44 => self.with_queue(0, |q| u32::from(q.ready)),
-                    0x60 => self.interrupt_status.load(Ordering::SeqCst),
+                    0x60 => {
+                        if !self.is_vhost_user {
```

Comment:
let's add a comment here similar to the one in the original commit + our current understanding of the issue we see with the original commit that led us to this approach.

Reply:
added a comment with explanation of vhost-user situation. But did not include the explanation of the reason why original approach did not work as we don't have enough data.